ad2games-ui_components.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
     'README.rdoc'
   ]
 
-  s.add_dependency 'rails', '~> 4.2.0'
```

Comment:
Don't we need to keep the `>= 4.2` restriction?

Reply:
Good point.